### DISCUSSION TITLE
高级模型（gpt-4）的潜在使用

### DISCUSSION THREAD

**Discussion by jiacheo** [langgenius/dify#3428] [General]
我在使用Dify构建各种类型的应用过程中，尽管在llm方面都设置了使用gpt-3.5-turbo， 但随着我通过各种设置、聊天之后，发现我的费用账单里突然出现了好几个gpt-4的费用，而且这些费用还不低。

这里是请教一个问题，是否在某些功能（比如instruction的automatic）上，项目是默认使用了gpt-4的（如果有设置openai的话），而不是遵循我们在llm层面选择的gpt-3.5-turbo（或者其他）。

如果有这种默认使用高级模型的情况，麻烦帮忙罗列一下，哪些场景会用到，让大家心里有数，不用担心某天聊天量变大，然后都走到类似gpt-4这种又贵又高级的模型上，从而导致账单的暴涨。

谢谢。

**Comment by crazywoola** [comment]
<img width="1093" alt="image" src="https://github.com/langgenius/dify/assets/100913391/b9fe6a38-a6e1-489c-811b-64c763f19209">
我猜应该是这个

**Reply by jiacheo** [reply]
没错！确实是这个，终于解决了gpt4突然用量增高的问题

**Reply by linfan2024** [reply]
终于知道了，莫名亏了好几块
